plasma_framework/test/src/exits/payment/PaymentPiggybackInFlightExitOnInput.test.js

```diff
@@ -0,0 +1,259 @@
+const ExitIdWrapper = artifacts.require('ExitIdWrapper');
+const OutputGuardParserRegistry = artifacts.require('OutputGuardParserRegistry');
+const PaymentInFlightExitRouter = artifacts.require('PaymentInFlightExitRouterMock');
+const PaymentStartInFlightExit = artifacts.require('PaymentStartInFlightExit');
+const PaymentSpendingConditionRegistry = artifacts.require('PaymentSpendingConditionRegistry');
+const PaymentPiggybackInFlightExitOnInput = artifacts.require('PaymentPiggybackInFlightExitOnInput');
+const PaymentPiggybackInFlightExitOnOutput = artifacts.require('PaymentPiggybackInFlightExitOnOutput');
+const SpyPlasmaFramework = artifacts.require('SpyPlasmaFrameworkForExitGame');
+
+const {
+    BN, constants, expectEvent, expectRevert, time,
+} = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { calculateNormalExitable } = require('../../../helpers/exitable.js');
+const { buildUtxoPos, utxoPosToTxPos } = require('../../../helpers/positions.js');
+const {
+    addressToOutputGuard,
+} = require('../../../helpers/utils.js');
+const { PaymentTransactionOutput, PaymentTransaction } = require('../../../helpers/transaction.js');
+
+contract('PaymentInFlightExitRouter', ([_, alice, bob]) => {
+    const PIGGYBACK_BOND = 31415926535; // wei
+    const ETH = constants.ZERO_ADDRESS;
+    const MIN_EXIT_PERIOD = 60 * 60 * 24 * 7; // 1 week
```

Comment:
I miss this all the time :nerd_face:. And all the time I read such is... yyy wait :joy: 
```suggestion
    const MIN_EXIT_PERIOD = 60 * 60 * 24 * 7; // 1 week in seconds
```